### DISCUSSION TITLE
How to follow Mastodon account?

### DISCUSSION THREAD

**Discussion by shigurenimo** [misskey-dev/misskey#9226] [Q&A] [answered]
I can't find `@uufish@pawoo.net` in mastodon.

![Screenshot 2022-11-24 at 22 43 31](https://user-images.githubusercontent.com/8127869/203799433-ab757bee-6a57-4b20-bf99-e0ed9842416f.png)

**Comment by shigurenimo** [comment, marked as answer]
ok, I had to click the search button.

**Comment by KawaneRio** [comment]
Please mark your question as closed if possible. Thank you: )

**Reply by tassoman** [reply]
Any project admin can't mark solutions?

**Reply by KawaneRio** [reply]
I think so. But I'm not an Admin here :shrug:
